eng/common/scripts/Verify-Links.ps1

```diff
@@ -167,6 +171,11 @@ function CheckLink ([System.Uri]$linkUri)
   }
   $checkedLinks[$linkUri] = $true;
 }
+$GithubRegex = "($sourceRepoUrl/blob/)([a-zA-Z0-9/-/_]+)(/.*)"
```

Comment:
Will match any branch, I think we likely only want to replace master links by default.

Reply:
What about feature branch?

Reply:
(just adding a comment to capture what we talked) As we talked about our guidelines are to use links to master so feature branches shouldn't be an issue here. Also as we are now passing in the regex we can control the branch outside of the script.